files/qsub.pl

```diff
@@ -289,6 +289,9 @@ sub make_command
     my @intcommand;
 
     if ($interactive || $fake) {
+        if ($script) {
+            fatal("Intreactive jobs are not allowed to run a jobscript (in this case: \"$script\")");
```

Comment:
`Interactive` (typo)

Reply:
Will fix the typos in the next PR, which will fix issue 17. (will submit soon)